src/cluster/cluster-registry.js

```diff
@@ -27,6 +27,7 @@ module.exports = class ClusterRegistry extends EventEmitter {
 	 */
 	constructor( options, connectionEndpoint ) {
 		super();
+		this.setMaxListeners( 20 );
```

Comment:
Yeah, that isn't the best approach since it will most likely also reach 20.

Can you out this in temporary:

``
process.on('warning', e => console.warn(e.stack));
```

and copy the stack trace?